JS/Lesson14/esra/script.js

```diff
@@ -115,12 +141,18 @@ console.log('Exercise11:', number10, number11);
 console.log('------------------------');
 
 // 12. Declare a variable number10 and initialize it to -3. Use the postfix ++ operator to increment the value of num by 1, then multiply the result by 2 and log it to the console.
+let number10 = -3;
+number10++;
+let result = number10++ * 2;
```

Comment:
In this exercise, you only need to increase the number by one once, and it's already done on the line 145. So here you only do the following:
 ```suggestion
let result = number10 * 2;
```